test/solution-template-validation-tests/test/mainTemplateTests.js

```diff
@@ -79,15 +79,16 @@ describe('template files - ', () => {
 
         });
 
-        /** The location parameter MUST NOT have allowedValues property and MUST have defaultValue property whose value MUST be '[resourceGroup().location]' */
+        /** The location parameter MUST NOT have allowedValues property and if it has defaultValue property whose value MUST be '[resourceGroup().location]' */
```

Comment:
The location parameter MUST NOT have allowedValues property and if it's value MUST be '[resourceGroup().location]'

Reply:
Here we need to check two properties, "allowedValues" and "defaultValue", so I think the original comment is more accurate.

Reply:
/** The location parameter MUST NOT have allowedValues property and if it has defaultValue property, it's value MUST be '[resourceGroup().location]' */